runners/spark/pom.xml

```diff
@@ -34,7 +34,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <spark.version>1.6.2</spark.version>
+    <spark.version>1.6.3</spark.version>
```

Comment:
Does the PR really require Spark 1.6.3 update (I don't think so). I would not mix different purpose in the same PR: Spark 1.6.3 upgrade should be in a dedicated PR IMHO.